types/fetch-mock/index.d.ts

```diff
@@ -156,13 +172,29 @@ declare namespace fetchMock {
          * if nothing matches it).
          */
         repeat?: number;
-    }
 
-    type MockCall = [string, MockRequest];
+        /**
+         * Convert objects into JSON before delivering as stub reponses. Can
+         * be useful to set to false globally if e.g. dealing with a lot of
+         * array buffers. If true, will also add content-type: application/json
+         * header.
+         * @default true
+         */
+        sendAsJson?: boolean;
 
-    interface MatchedRoutes {
-        matched: MockCall[];
-        unmatched: MockCall[];
+        /**
+         * Automatically sets a content-length header on each response.
+         * @default true
+         */
+        includeContentLength?: boolean;
+    }
+
+    interface MockCall extends Array<string | RequestInit | undefined> {
```

Comment:
Have a look at https://github.com/wheresrhys/fetch-mock/blob/master/src/lib/fetch-handler.js#L136

MockCall is an array [url, options].  Three properties are added to this:
* identifier
* isUnmatched
* request

"request" is the only one of these mentioned in the documentation, it says "If fetch was called using a Request instance, this will be available as a request property on this array"
http://www.wheresrhys.co.uk/fetch-mock/#api-inspectioncalls